tools/linter/adapters/s3_init_config.json

```diff
@@ -21,17 +21,13 @@
         }
     },
     "clang-tidy": {
-        "Darwin-i386": {
-            "download_url": "https://oss-clang-format.s3.us-east-2.amazonaws.com/macos-i386/15.0.6/clang-tidy",
-            "hash": "11c9234155dd5b7aec8cf46ea9629401c4432576615b6eff2a5a4c5e3f9e6504"
-        },
```

Comment:
The binaries is still available, I see no reason to remove it. Just properly update this to the latest version too. There are use cases where someone may want to run to clang-tidy / clang-format on an intel macbook even if we don't officially support it.